README.md

```diff
@@ -1,148 +1,7 @@
-* `PASS_CORE_LOGOUT_DELETE_COOKIES="JSESSIONID /,shib_idp_session /idp"` : Cookies to delete on logout, "name path" separated by commas.
-* `POSTGRES_USER=postgres`
-* `POSTGRES_PASSWORD=postgres`
-* `JDBC_DATABASE_URL=jdbc:postgresql://postgres:5432/pass`
-* `JDBC_DATABASE_USERNAME=pass`
-* `JDBC_DATABASE_PASSWORD=moo`
-
-### `postgres`
-
-Pretty much an out-of-the-box PostgreSQL server. Only interacts with the [`pass-core`](https://github.com/eclipse-pass/pass-core) service.
-
-### [`pass-ui`](https://github.com/eclipse-pass/pass-ui)
-
-Repository: https://github.com/eclipse-pass/pass-ui
-Package: https://github.com/orgs/eclipse-pass/packages/container/package/pass-ui
-
-User interface for the PASS application. Currently does not handle environment variables nicely - they are baked into images at build time due. The environment variables in the demo environment should not need to be adjusted between different deployment environments.
-
-Environment variables:
-
-* `PASS_UI_PORT=81`
-* `PASS_API_NAMESPACE=data`
-* `PASS_UI_ROOT_URL=/app`
-* `STATIC_CONFIG_URL=/app/config.json`
-* `DOI_SERVICE_URL=/doiservice/journal`
-* `MANUSCRIPT_SERVICE_LOOKUP_URL=/downloadservice/lookup`
-* `MANUSCRIPT_SERVICE_DOWNLOAD_URL=/downloadservice/download`
-* `POLICY_SERVICE_POLICY_ENDPOINT=/policyservice/policies`
-* `POLICY_SERVICE_REPOSITORY_ENDPOINT=/policyservice/repositories`
-* `SCHEMA_SERVICE_URL=/schemaservice`
-* `USER_SERVICE_URL=/pass-user-service/whoami`
-
-
-### `loader`
-
-A basic Docker image where we can run a `curl` command to bootstrap the environment with data from `demo_data.json`
-
-## Running Acceptance Tests
-
-Repository: https://github.com/eclipse-pass/pass-acceptance-testing
-
-There is a small set of end-to-end smoke tests that we can run against this environment for some validation of changes. These tests run automatically for new PRs that are opened against `main`, but can also be run locally. In order to do this, you'll first need to clone the repository with the tests.
-
-Once you have the repository, wait for the Docker Compose environment to start up and initialize then run the tests directly. From the `pass-acceptance-testing` local repo, run:
-
-``` sh
-yarn            # Installs project dependencies
-yarn run test   # Runs tests
-```
+PASS Docker is a developer-focused PASS runtime, which provides the PASS project and all of its dependent services using
+[Docker Compose](https://docs.docker.com/compose/). PASS Docker provides Docker images that can be used for running PASS
+in different environments including a local test instance and production.
 
+You can find further documentation and details about [PASS Docker](https://docs.eclipse-pass.org/developer-documentation/pass-docker)
+on the [full documentation site](https://docs.eclipse-pass.org). A full list of all the Eclipse PASS projects can also
+be found in the PASS Main repository [README](https://github.com/eclipse-pass/main/blob/main/README.md).
```

Comment:
I don't think this should be the link. It should be at the top level like https://github.com/eclipse-pass/main

Reply:
@markpatton This setup was from what we discussed. See this [comment](https://docs.google.com/document/d/1tIdyNnODtHxACHTYvFDAzH4mELgT3lOm-cJ88UKfO30/edit?disco=AAABYEfDLhE).

Do you want to discuss this further?

Reply:
Ah I think I see what you mean, you'd prefer it to land on the main page that displays the README at the bottom instead of a direct link to the README itself.

@bbranan What do you think?

Reply:
Done.